kokoro/macos/build.sh

```diff
@@ -23,7 +23,7 @@ SRC=$PWD/github/agi/
 curl -L -k -O -s https://dl.google.com/android/repository/tools_r25.2.3-macosx.zip
 mkdir android
 unzip -q tools_r25.2.3-macosx.zip -d android
-echo y | ./android/tools/bin/sdkmanager build-tools\;29.0.2 platforms\;android-26
+echo y | ./android/tools/bin/sdkmanager build-tools\;30.0.3 platforms\;android-26
```

Comment:
I think you might need to install android-30 here:

```suggestion
echo y | ./android/tools/bin/sdkmanager build-tools\;30.0.3 platforms\;android-30
```

Idem on other platforms